packages/core/content-manager/admin/src/pages/App/tests/index.test.js

```diff
@@ -0,0 +1,474 @@
+/* eslint-disable no-irregular-whitespace */
+import React from 'react';
+import { createStore, combineReducers } from 'redux';
+import { Router } from 'react-router-dom';
+import { createMemoryHistory } from 'history';
+import { Provider } from 'react-redux';
+import { render, screen } from '@testing-library/react';
+import ContentManagerApp from '..';
+import cmReducers from '../../../reducers';
+import pluginId from '../../../pluginId';
+import useModels from '../useModels';
+
+jest.mock('../useModels', () =>
+  jest.fn(() => {
+    return {};
+  })
+);
+
+jest.mock('react-intl', () => ({
+  FormattedMessage: () => 'label',
+  useIntl: () => ({ formatMessage: jest.fn(() => 'label') }),
+}));
+jest.mock('@fortawesome/react-fontawesome', () => ({
+  FontAwesomeIcon: () => null,
+}));
+
+describe('Content manager | App | main', () => {
+  beforeEach(() => {
+    jest.resetModules(); // Most important - it clears the cache
+  });
+
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it('should not crash', () => {
+    const contentManagerState = {
+      collectionTypeLinks: [
+        {
+          uid: 'category',
+          title: 'Categories',
+          name: 'category',
+          to: '/category',
+          kind: 'collectionType',
+          isDisplayed: true,
+          permissions: [
+            {
+              action: 'plugins::content-manager.explorer.read',
+              subject: 'category',
+            },
+          ],
+        },
+      ],
+      singleTypeLinks: [
+        {
+          uid: 'homepage',
+          title: 'Home page',
+          name: 'homepage',
+          to: '/homepage',
+          kind: 'singleType',
+          isDisplayed: true,
+          permissions: [
+            {
+              action: 'plugins::content-manager.explorer.read',
+              subject: 'homepage',
+            },
+          ],
+        },
+      ],
+      models: [
+        {
+          kind: 'collectionType',
+          uid: 'category',
+          info: { label: 'Categories', name: 'category' },
+        },
+        { kind: 'singleType', uid: 'homepage', info: { label: 'Home page', name: 'homepage' } },
+      ],
+      components: [],
+      status: 'resolved',
+    };
+    useModels.mockImplementation(() => contentManagerState);
+    const rootReducer = combineReducers(cmReducers);
+    const store = createStore(rootReducer, { [`${pluginId}_app`]: contentManagerState });
+    const history = createMemoryHistory();
+    history.push('/plugins/content-manager/collectionType/category');
```

Comment:
Can you change the initial location to `/plugins/content-manager` to make sure the App redirects to the category page?